float32[] x
float32[] y
---
std_msgs/Float32MultiArray[] value
